common/remote_util.h

```diff
@@ -101,7 +106,7 @@ class RemoteUtil {
   bool Quiet() const { return quiet_; }
 
  private:
-  // Verifies that both |gamelet_ip_| and |ssh_port_| are set.
+  // Verifies that both |hostname_| and |ssh_port_| are set.
   absl::Status CheckIpPort();
```

Comment:
Nit: `CheckHostPort()`

Reply:
Done. Btw. since SetHostPort is now the only way to get it into a proper state, we might as well take that as constructor args.